docs/README.md

```diff
@@ -157,22 +157,6 @@ kotlin {
 }
 ```
 
-### Support Android API < 26
-
-If your min SDK version is below 26, you have to enable library desugaring as described in the [Android documentation](https://developer.android.com/studio/write/java8-support#library-desugaring):
-
-```kotlin
-android {
-    compileOptions {
-        isCoreLibraryDesugaringEnabled = true
-    }
-}
-
-dependencies {
-    coreLibraryDesugaring("com.android.tools:desugar_jdk_libs:2.1.5")
-}
-```
-
```

Comment:
I'm pretty sure we have to keep that as our min SDK is below 26 isn't it?

Reply:
I think we only added that because of `kotlinx.datetime`. But that library is only used in the demo apps now, so there's no need for integrators to do that.
If you think it is relevant to keep it for other reasons, please let me know, and I'll add it back.

Reply:
If integrators doesn't need that to use our library with api < 26, I'm fine to remove it.